javaagent-log-appender/otel-config.yaml

```diff
@@ -0,0 +1,22 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
```

Comment:
Lines 4 and 5 don't seem necessary

Reply:
They are needed as I opted to provide optional support for `grpc` protocol https://github.com/open-telemetry/opentelemetry-java-examples/pull/503/files#diff-1b2f929f45fbea4950cb70b3a03c0c515e5821085c60c298d741be2822967f9eR53-R56